ui/selectors/assets.ts

```diff
@@ -232,9 +235,9 @@ export const getMultiChainAssets = createDeepEqualSelector(
           chainId,
           isNative,
           primary: balance.amount,
-          secondary: balanceInFiat.toNumber(),
+          secondary: balanceInFiat ? balanceInFiat.toNumber() : null,
           string: '',
-          tokenFiatAmount: balanceInFiat.toNumber(), // for now we are keeping this is to satisfy sort, this should be fiat amount
+          tokenFiatAmount: balanceInFiat ? balanceInFiat.toNumber() : null,
```

Comment:
Can this be just:
`tokenFiatAmount: balanceInFiat;`
I think no need to add the null fallback since we are already doing it here
```
const balanceInFiat = rate
        ? new BigNumber(balance.amount).times(rate)
        : null;
```

Reply:
Yeah, fair point. I've updated here; https://github.com/MetaMask/metamask-extension/pull/32731/commits/5cdd42fdc0768c8c5308e805ae421ef2b1ef6303